app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
```

Comment:
Let's talk about this a bit more at/after standup.  I agree with you, but I want to think of the best way to handle this.